### PR TITLE
profiling info tests not instantiated

### PR DESCRIPTION
**Pull request by songbell**
### Details:
 - fix tests not enabled in ci tests

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/auto_batch/tests/functional/skip_tests_config.cpp [hunk 1]
```diff
@@ -24,6 +24,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*VirtualPlugin.*BehaviorTests.*OVHoldersTest.*)",
         // BATCH plugin doesn't support this case
         R"(.*LoadNetworkCreateDefaultExecGraphResult.*)",
+        // BATCH/TEMPLATE plugin doesn't support this case
+        R"(.*OVInferRequestPerfCountersTest.*CheckOperationInProfilingInfo.*)",
```
Comment (ilya-lavrenov) [inline src/plugins/auto_batch/tests/functional/skip_tests_config.cpp:28]: can we implement / emulate this in template plugin?
CC @ilyachur

Comment (ilyachur) [reply src/plugins/auto_batch/tests/functional/skip_tests_config.cpp:28]: Yes we can.
Why auto batch mentions template plugin?

Comment (ilya-lavrenov) [reply src/plugins/auto_batch/tests/functional/skip_tests_config.cpp:28]: Looks like only in tests.

Comment (songbell) [reply src/plugins/auto_batch/tests/functional/skip_tests_config.cpp:28]: @riverlijunjie  I think here batch is leverage template to cover more tests, am I right?

**Review by ilyachur — APPROVED** [review]

**Pull request merged**